### PR TITLE
feat(Image): support reset action in toolbarRender

### LINKED ISSUE #48922: Image 组件中 toolbarRender 的 info 参数里面的 actions，现在只暴露出了 onZoomOut 和 onZoomIn，期望能有一个 重置action
### What problem does this feature solve?

图片放大或者缩小之后可以通过一个按钮点击达到初始化的预览效果

### What does the proposed API look like?

和 onZoomOut 同级，多暴露出一个方法 比如 onReset，或者onZoomOut 添加参数可以达到自定义缩小多少倍

### PR DESCRIPTION
**Pull request by kiner-tang**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close #48922

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Image support reset action in toolbarRender |
| 🇨🇳 Chinese |  图片组件的工具栏渲染函数中支持重置方法  |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
feature branch

File: components/image/index.zh-CN.md [hunk 1]
```diff
@@ -157,6 +158,8 @@ type TransformAction =
     onRotateRight: () => void;
     onZoomOut: () => void;
     onZoomIn: () => void;
+    onReset: () => void;// 5.17.3 之后支持
```
Comment (afc163) [inline components/image/index.zh-CN.md:161]:
```suggestion
    onReset: () => void; // 5.17.3 之后支持
```

**Comment by afc163** [comment]
为啥 https://preview-48936-ant-design.surge.sh/components/image-cn/ 这里没看到这个按钮。

**Comment by kiner-tang** [comment]
> 为啥 https://preview-48936-ant-design.surge.sh/components/image-cn/ 这里没看到这个按钮。

在自定义工具栏这里：

image

**Review by afc163 — APPROVED** [review]

**Pull request merged**